js/apps/system/_admin/aardvark/APP/react/package.json

```diff
@@ -95,104 +83,24 @@
-    "@svgr/webpack": "2.4.1",
-    "@types/backbone": "^1.4.3",
-    "@types/d3": "^5.5.0",
-    "@types/jquery": "^3.5.0",
-    "@types/jsoneditor": "^5.28.2",
-    "@types/lodash": "^4.14.172",
-    "@types/nvd3": "^1.8.41",
-    "@types/react": "^16.9.38",
-    "@types/react-dom": "^16.9.8",
-    "@types/react-redux": "^6.0.12",
-    "@types/react-router-dom": "5.3.1",
-    "@types/sigmajs": "^1.0.27",
-    "@types/styled-components": "^4.4.3",
-    "@types/underscore": "^1.10.1",
-    "@typescript-eslint/eslint-plugin": "^4.0.0",
-    "@typescript-eslint/parser": "^4.0.0",
-    "babel-eslint": "^10.0.0",
-    "babel-jest": "^27.0.6",
-    "babel-loader": "8.0.4",
-    "babel-plugin-named-asset-import": "^0.2.3",
-    "babel-preset-react-app": "^6.1.0",
-    "case-sensitive-paths-webpack-plugin": "2.1.2",
-    "compression-webpack-plugin": "^4.0.0",
-    "eslint": "^7.31.0",
-    "eslint-config-react-app": "^6.0.0",
-    "eslint-loader": "^4.0.2",
-    "eslint-plugin-flowtype": "^5.2.0",
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-jsx-a11y": "^6.3.1",
-    "eslint-plugin-react": "^7.20.3",
-    "eslint-plugin-react-hooks": "^4.0.8",
-    "fork-ts-checker-webpack-plugin-alt": "0.4.14",
-    "html-webpack-plugin": "4.0.0-alpha.2",
-    "jest": "^25.5.4",
-    "jest-pnp-resolver": "1.0.1",
-    "jest-resolve": "23.6.0",
-    "optimize-css-assets-webpack-plugin": "5.0.1",
-    "pnp-webpack-plugin": "1.1.0",
-    "react-dev-utils": "^6.1.1",
-    "sass": "^1.58.3",
-    "terser-webpack-plugin": "^3.0.6",
-    "typescript": "^4.3.5",
-    "webpack": "^4.43.0",
-    "webpack-concat-plugin": "^3.0.0",
-    "webpack-dev-server": "^3.11.0",
-    "webpack-manifest-plugin": "2.0.4",
-    "workbox-webpack-plugin": "3.6.3"
-  },
-  "jest": {
-    "collectCoverageFrom": [
-      "src/**/*.{js,jsx,ts,tsx}",
-      "!src/**/*.d.ts"
-    ],
-    "resolver": "jest-pnp-resolver",
-    "setupFiles": [
-      "react-app-polyfill/jsdom"
-    ],
-    "testMatch": [
-      "<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}",
-      "<rootDir>/src/**/?(*.)(spec|test).{js,jsx,ts,tsx}"
-    ],
-    "testEnvironment": "jsdom",
-    "testURL": "http://localhost",
-    "transform": {
-      "^.+\\.(js|jsx|ts|tsx)$": "<rootDir>/node_modules/babel-jest",
-      "^.+\\.css$": "<rootDir>/config/jest/cssTransform.js",
-      "^(?!.*\\.(js|jsx|ts|tsx|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
-    },
-    "transformIgnorePatterns": [
-      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|ts|tsx)$",
-      "^.+\\.module\\.(css|sass|scss)$"
-    ],
-    "moduleNameMapper": {
-      "^react-native$": "react-native-web",
-      "^.+\\.module\\.(css|sass|scss)$": "identity-obj-proxy"
-    },
-    "moduleFileExtensions": [
-      "web.js",
-      "js",
-      "web.ts",
-      "ts",
-      "web.tsx",
-      "tsx",
-      "json",
-      "web.jsx",
-      "jsx",
-      "node"
-    ]
+    "@craco/craco": "^7.1.0",
```

Comment:
Apologies, I meant to pin it to whatever it was already on. As the yarn.lock was gone I couldn't see it, just checked, we should pin to 7.8.0 because that's what yarn was resolving to.

Reply:
Sorry this comment was supposed to be arangojs. Something got messed up while reviewing